hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -44,6 +49,7 @@
 public class HiveCatalog extends BaseMetastoreCatalog implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(HiveCatalog.class);
 
+
```

Comment:
Please remove non-functional changes like this one.

Reply:
thanks Ryan , now has fixed it.